src/Commands/GetEntryPoints.ts

```diff
@@ -0,0 +1,69 @@
+import { EntryPoint, EntryPoints, RestProjectsOptions } from '../EntryPoint';
+import { logger } from '../Utils';
+import { Arguments, Argv, CommandModule } from 'yargs';
+import { container } from 'tsyringe';
+
+export class GetEntryPoints implements CommandModule {
+  public readonly command = 'get-entrypoints [options]';
```

Comment:
Let's use either the composite commands or existing convention to define a command:

### Composite Commands Approach:

#### Example
```ts
export class GetEntrypoints implements CommandModule {
  public readonly command = 'get [options]';
  // ...
}

export class Entrypoints implements CommandModule {
  public readonly command = 'entrypoints [options]';
  // ...

  public builder(yargs: Argv): Argv<EntrypointsParams> {
    return yargs
      // ...
      .command<GetEntrypointsParams>(new GetEntrypoints())
      .demandCommand(1);
  }

  public handler(_: Arguments<EntrypointsParams>): void {}
}
```

#### Usage

```sh
bright-cli entrypoints get [options]
```

### Existing Convention:

#### Example
```ts
export class GetEntrypoints implements CommandModule {
  public readonly command = 'entrypoint:list [options]';
  // ...
}
```

#### Usage

```sh
bright-cli entrypoint:list [options]
```

Please discuss with product team to determine the most appropriate option. However, it is preferable to align with existing convention.

Reply:
sure, I like the existing approach.

Reply:
I applied the 2nd, and waiting for approval from product team.